LongWarOfTheChosen/Localization/LW_Overhaul.rus

```diff
@@ -612,3 +612,23 @@ m_strNextHaven="СЛЕД. УБЕЖИЩЕ"
 [Logistics X2LWAlienActivityTemplate]
 ActivityObjectives[0]="Захватить набор ресурсов \"Адвента\""
 +MissionDescriptions=(MissionFamily="SmashnGrab_LW", Description="Сопротивление обнаружило расположение ценных ресурсов противника которые мы можем заполучить для своих потребностей.", MissionIndex=0)
+
+; LWOTC Translated
+ActivityObjectives[0]="Начало освобождения региона"
+UpToText="До"
+ExpectedActivityTextStr="Ожидаемая активность"
+BoostedActivityText="Ожидаемая активность (с ускорением)"
+BoostTextPre="С"
+BoostTextPost="Ускорение"
+; End Translated
+
+; LWOTC Translated
+strAreaOfOperations="Местность операции:"
+; End Translated
```

Comment:
And this should be moved to the section with the header `[UIScreenListener_SquadSelect_LW]`. Let me know if you don't understand what I mean.

Reply:
done